src/config/feature-flags.ts

```diff
@@ -26,6 +26,8 @@ export enum BooleanFlags {
 	PRIORITIZE_PUSHES = "prioritize-pushes",
 	USE_NEW_GITHUB_CLIENT__FOR_PR = "git-hub-client-for-pullrequests",
 	NEW_REPO_SYNC_STATE = "new-repo-sync-state",
+	SUPPORT_BRANCH_AND_MERGE_WORKFLOWS_FOR_DEPLOYMENTS = "support-branch-and-merge-workflows-for-deployments",
+	INCLUDE_JIRA_HOSTNAME_TO_DEDUP_KEY = "include-jira-hostname-to-dedup-key",
```

Comment:
I think this feature flag has been removed before (probably came in due to a merge conflict).